conans/model/version_range.py

```diff
@@ -96,9 +101,10 @@ def __init__(self, expression):
     def __str__(self):
         return self._expression
 
-    def __contains__(self, version):
+    def contains(self, version, conf_resolve_prerelease=None):
```

Comment:
I'd say this default is not the safest, we might skip some ``version in range`` occurrence that needs the conf passed, and we are not passing it. I'd suggest dropping the ``=None`` default and having this forced, even if it needs changing more tests.